src/domains/chart/components/portals.tsx

```diff
@@ -7,7 +7,7 @@ import { DisableOutOfView } from "./disable-out-of-view"
 
 const getNodesArray = () => Array.from(document.querySelectorAll("[data-netdata]"))
 
-export const Portals = () => {
+export const Portals = memo(() => {
```

Comment:
So, it doesn't have any dependencies ever to re-consider the memo?

Reply:
I don't understand, do you mean that it doesn't use any Props? This component will not have to update based on parent state.